components/PlantDetail.js

```diff
@@ -67,6 +69,15 @@ export default function PlantDetail({
           handleCancel={handleCancel}
         />
       )}
+      <NotesContainer>
+        <FormComments
+          onSubmit={handleAddNotes}
```

Comment:
```suggestion
          onSubmit={onAddNotes}
```
It it a common practice to use the Prefix "on" up until the point where the function is actually defined. :)